docker/lookaside_test.go

```diff
@@ -28,13 +29,21 @@ func TestConfiguredSignatureStorageBase(t *testing.T) {
 	assert.Error(t, err)
 
 	// No match found
+	// get default storage base
 	emptyDir, err := ioutil.TempDir("", "empty-dir")
 	require.NoError(t, err)
 	defer os.RemoveAll(emptyDir)
 	base, err := configuredSignatureStorageBase(&types.SystemContext{RegistriesDirPath: emptyDir},
 		dockerRefFromString(t, "//this/is/not/in/the:configuration"), false)
+	expectBase := "file://"
+	if rootless.GetRootlessEUID() == 0 {
+		expectBase += filepath.Join(defaultDockerDir, "//this/is/not/in/the")
+	} else {
+		expectBase += filepath.Join(os.Getenv("HOME"), defaultUserDockerDir, "//this/is/not/in/the")
+	}
```

Comment:
No, please test both: split `configuredSignatureStorageBaseForEUID` from `configuredSignatureStorageBase`, document that it only exists for tests, and test the `…ForEUID` variant here, for both the root and non-root cases. Or maybe split a `builtinDefaultSignatureStorageDir` with an EUID parameter, test both only for that one small helper, and here assume that tests are not running as root.